mods/content/psionics/system/psionics/equipment/psipower.dm

```diff
@@ -13,6 +13,7 @@
 	abstract_type = /obj/item/psychic_power
 
 	var/maintain_cost = 3
+	var/backblast_on_failed_maintain = FALSE
```

Comment:
Where is this used?

Reply:
I feel like maybe it's supposed to be passed as `backblast_on_failure` later in `/Process()`?